recipes/lammps/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "lammps" %}
+{% set version = "stable_16Mar2018" %}
+{% set sha256 = "03f91d69f472c37a4008611dee13fad91b6bedc66dbf0393416cdacc751b57ff" %}
+{% set date = datetime.datetime.strptime(version.split('_')[1], "%d%b%Y") %}
+{% set conda_version = "{:%Y.%m.%d}".format(date) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ conda_version }}
+
+source:
+  fn: {{ name }}-{{ conda_version }}.tar.gz
+  url: https://github.com/lammps/lammps/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+  - {{ compiler('c') }}
+  - {{ compiler('cxx') }}
+  - {{ compiler('fortran') }}
+  
+  host:
+    - python
+    - setuptools
+    - cmake
+    - make 
+    - mpich
```

Comment:
Would this maybe work with `openmpi` also? We could set it up to support either version if so (using https://github.com/conda-forge/staged-recipes/pull/1501).

Also, since this is apparently only used on Linux, we should only require it there:
```
    - mpich  # [linux]
```